server/api/routes/validators/users.py

```diff
@@ -0,0 +1,45 @@
+from email_validator import validate_email, EmailNotValidError
+
+def valid_email(email):
+    """Checks if an email is valid
+
+    Args:
+        email (str): user's email address
+
+    Returns:
+        (True, email): normalized email if it is valid
+        (False, error): error message if it is invalid
+    """
+    try:
+        valid = validate_email(email)
+        email = valid.email
+    except EmailNotValidError as e:
+        return (False, str(e))
+
+    return (True, email)
+
+def valid_password(password):
+    """Checks if a password is valid
+
+    Args:
+        password (string): user's password
+
+    Returns:
+        (True, password): password if it is valid
+        (False, error): error message if it is invalid
+    """
+
+    # Is at least 8 characters long
+    if len(password) < 8:
+        return (False, 'The password is not valid. It must have at least 8 characters.')
```

Comment:
You can probably shorten these to something like "Password must contain at least 8 characters"

Reply:
Will do. Only reason I did that was because `email-validator` returns errors in that format.